dist/fixto.js

```diff
@@ -260,7 +258,7 @@ var fixto = (function ($, window, document) {
     // Checks if browser creates a positioning context for fixed elements.
     // Transform rule will create a positioning context on browsers who follow the spec.
     // Ie for example will fix it according to documentElement
-    // TODO: Other css rules also effects. perspective creates at chrome but not in firefox. transform-style preserve3d effects.
```

Comment:
preserve3d was not a typo, it is a css value for transfom-value attribute but actually should be preserve-3d.